hack/update-bindata.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+KUBE_ROOT="../../../"
+if ! [ -d $KUBE_ROOT/examples ]; then
+	echo "This is meant to be run from the test/e2e/ directory by test/e2e/e2e.go or similar."
+	exit 1
+fi
+echo "running generation framework"
+pwd
```

Comment:
rather than relying on the working directory to be correct, maybe we should be explicit about where the output is saved?